### ISSUE TITLE
Redirect paths for SSO login should be exposed as variable

### ISSUE THREAD

**Issue by wtrocki** [redhat-developer/app-services-cli#787]
See: 
https://github.com/redhat-developer/app-services-cli/blob/853c904858deb2164e6c72ff60e1b4a3b9a0cf65/pkg/cmd/login/login.go#L169

**Comment by craicoverflow** [comment]
Yes. They should go here: https://github.com/redhat-developer/app-services-cli/blob/main/internal/build/build.go#L35
